src/App.tsx

```diff
@@ -1,61 +1,112 @@
-import './App.scss';
+/*eslint-disable*/
```

Comment:
Don't disable `eslint`. This tool should force a good code style. Without it, keeping your code clean and readable would be more problematic.